src/applications/hca/containers/IntroductionPageGated.jsx

```diff
@@ -55,8 +62,30 @@ const VerificationRequiredAlert = () => (
   />
 );
 
-const LoggedOutContent = ({ route }) => (
+const LoggedOutContent = connect(
```

Comment:
I turned LoggedOutContent into a connected component so it has access to the `toggleLoginModal` action creator, rather than adding the action creator to the main outer component and passing it down via props.